packs/boundary_worker/README.md

```diff
@@ -0,0 +1,126 @@
+| nomad_group_count                 | Count of Deployments for the Group.          | `1` |
+| nomad_group_ephemeral_disk        | Ephemeral Disk Configuration for the Group.  | `{"migrate":true,"size":128,"sticky":true}` |
+| nomad_group_name                  | Name for the Group.                          | `"boundary_worker"` |
+| nomad_group_network_mode          | Network Mode for the Group.                  | `"host"` |
+| nomad_group_ports                 | Port Configuration for the Group.            | `{"ops":{"check_interval":"30s","check_timeout":"15s","host_network":null,"name":"boundary_worker_ops","path":"/health","port":9203,"protocol":"http","type":"http"},"proxy":{"check_interval":"30s","check_timeout":"15s","host_network":null,"name":"boundary_worker_proxy","path":null,"port":9202,"protocol":"tcp","type":"tcp"}}` |
+| nomad_group_restart_logic         | Restart Logic for the Group.                 | `{"attempts":3,"delay":"30s","interval":"120s","mode":"fail"}` |
+| nomad_group_service_name_prefix   | Name of the Service for the Group.           | `"boundary_worker"` |
+| nomad_group_service_provider      | Provider of the Service for the Group.       | `"nomad"` |
+| nomad_group_tags                  | List of Tags for the Group.                  | `["boundary","boundary-workers"]` |
+| nomad_group_volumes               | Volumes for the Group.                       | `{}` |
+| nomad_job_datacenters             | Eligible Datacenters for the Job.            | `["*"]` |
+| nomad_job_name                    | Name for the Job.                            | `"boundary_worker"` |
+| nomad_job_namespace               | Namespace for the Job.                       | `"default"` |
+| nomad_job_priority                | Priority for the Job.                        | `10` |
+| nomad_job_region                  | Region for the Job.                          | `"global"` |
+| nomad_pack_verbose_output         | Toggle to enable verbose output.             | `true` |
+| nomad_task_driver                 | Driver to use for the Task.                  | `"raw_exec"` |
+| nomad_task_name                   | Name for the Task.                           | `"boundary_worker"` |
+| nomad_task_resources              | Resource Limits for the Task.                | `{"cores":null,"cpu":500,"memory":512,"memory_max":1024}` |
+<!-- END_PACK_DOCS -->
+
+### Outputs
+
+For outputs, see [./outputs.tpl](./outputs.tpl).
+
+> **Note**
+>
+> The outputs are only rendered if `nomad_pack_verbose_output` is set to `true`.
+
+## Notes
+
+### Resource Sizing
+
+For hardware sizing of the underlying Nomad Agents that are designed to run the Boundary Worker in this Nomad Pack, see the [HCP Boundary Self-managed Worker sizing guidelines](https://developer.hashicorp.com/hcp/docs/boundary/self-managed-workers/size-self-managed-workers#sizing-guidelines-for-self-managed-workers).
```

Comment:
Aren't the default resource limits above too small compared to the sizing guidelines?

Reply:
The sizing guidelines definitely call for more memory and CPU than specified here. 

I think as defaults, these values are fine, but should definitely be tweaked by specifying a more appropriate value when running the Pack.

AWS M5 instances are 8GB of memory, which this Pack does _not_ need, and setting it as the default would make it harder to test it.